drivers/include/ina220.h

```diff
@@ -0,0 +1,195 @@
+    INA220_BRNG_16V_FSR = 0x0000, /**< 16 V bus voltage full scale range */
+    INA220_BRNG_32V_FSR = 0x0200, /**< 32 V bus voltage full scale range, default. */
+} ina220_brng_t;
+
+/**
+ * @brief Shunt ADC settings
+ *
+ * @see Table 5 in INA220 data sheet
+ */
+typedef enum ina220_sadc {
+    INA220_SADC_9BIT            = 0x0000, /**< 9 bit resolution, 84 us conversion time */
+    INA220_SADC_10BIT           = 0x0008, /**< 10 bit resolution, 148 us conversion time */
+    INA220_SADC_11BIT           = 0x0010, /**< 11 bit resolution, 276 us conversion time */
+    INA220_SADC_12BIT           = 0x0018, /**< 12 bit resolution, 532 us conversion time, default */
+    INA220_SADC_AVG_1_SAMPLE    = 0x0040, /**< 12 bit resolution, 532 us conversion time, same as INA220_SADC_12BIT */
+    INA220_SADC_AVG_2_SAMPLES   = 0x0048, /**< 2 sample average, 1.06 ms conversion time */
+    INA220_SADC_AVG_4_SAMPLES   = 0x0050, /**< 4 sample average, 2.13 ms conversion time */
+    INA220_SADC_AVG_8_SAMPLES   = 0x0058, /**< 8 sample average, 4.26 ms conversion time */
+    INA220_SADC_AVG_16_SAMPLES  = 0x0060, /**< 16 sample average, 8.51 ms conversion time */
+    INA220_SADC_AVG_32_SAMPLES  = 0x0068, /**< 32 sample average, 17.02 ms conversion time */
+    INA220_SADC_AVG_64_SAMPLES  = 0x0070, /**< 64 sample average, 34.05 ms conversion time */
+    INA220_SADC_AVG_128_SAMPLES = 0x0078, /**< 128 sample average, 68.10 ms conversion time */
+} ina220_sadc_t;
+
+/**
+ * @brief Bus ADC settings
+ *
+ * @see Table 5 in INA220 data sheet
+ */
+typedef enum ina220_badc {
+    INA220_BADC_9BIT            = 0x0000, /**< 9 bit resolution, 84 us conversion time */
+    INA220_BADC_10BIT           = 0x0080, /**< 10 bit resolution, 148 us conversion time */
+    INA220_BADC_11BIT           = 0x0100, /**< 11 bit resolution, 276 us conversion time */
+    INA220_BADC_12BIT           = 0x0180, /**< 12 bit resolution, 532 us conversion time, default */
+    INA220_BADC_AVG_1_SAMPLE    = 0x0400, /**< 12 bit resolution, 532 us conversion time, same as INA220_BADC_12BIT */
+    INA220_BADC_AVG_2_SAMPLES   = 0x0480, /**< 2 sample average, 1.06 ms conversion time */
+    INA220_BADC_AVG_4_SAMPLES   = 0x0500, /**< 4 sample average, 2.13 ms conversion time */
+    INA220_BADC_AVG_8_SAMPLES   = 0x0580, /**< 8 sample average, 4.26 ms conversion time */
+    INA220_BADC_AVG_16_SAMPLES  = 0x0600, /**< 16 sample average, 8.51 ms conversion time */
+    INA220_BADC_AVG_32_SAMPLES  = 0x0680, /**< 32 sample average, 17.02 ms conversion time */
+    INA220_BADC_AVG_64_SAMPLES  = 0x0700, /**< 64 sample average, 34.05 ms conversion time */
+    INA220_BADC_AVG_128_SAMPLES = 0x0780, /**< 128 sample average, 68.10 ms conversion time */
+} ina220_badc_t;
+
+/** INA220 reset command bit (in configuration register) */
+#define INA220_RESET_BIT (0x8000)
+
+/**
+ * @brief Initialize a current sensor
+ *
+ * @param[out] dev          device descriptor of sensor to initialize
```

Comment:
To be honest I don't have a concrete rule for this. Generally I name parameters as `out` when I assume to get some kind of data back from a function (~"function output"). When I want to init or send something, I do understand this as an input for the function and name it `in` (~"function input") .